ansible/roles/commcarehq/templates/logrotate.conf.j2

```diff
@@ -0,0 +1,29 @@
+# see "man logrotate" for details
+# rotate log files weekly
+weekly
+
+# packages drop log rotation information into this directory
+include /etc/logrotate.d
+
+# no packages own wtmp, or btmp -- we'll rotate them here
+/var/log/wtmp {
+    missingok
+    monthly
+    create 0664 root utmp
+    rotate 1
+}
+
+/var/log/btmp {
+    missingok
+    monthly
+    create 0660 root utmp
+    rotate 1
+}
+
+{{ log_home }}/*.log {
+    weekly
+    missingok
+    compress
+    delaycompress # compress on second rotation
+    olddir {{ rotated_log_home }}
```

Comment:
ah was just about to mention the that they were gzipped. `zgrep` sounds cool. i did do it on purpose because i've seen other places that are littered with log rotates that it's hard to find what real files are there